src/main/resources/jsons/carsToFix.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "id": 1642352660088,
+    "registrationNumber": "L0018",
```

Comment:
Maybe we should add this path with json files to the .gitignore, then we do not pollute the repo 😉